Prac_08/taxi.py

```diff
@@ -0,0 +1,36 @@
+"""
+CP1404/CP5632 Practical
+Taxi class
+"""
+from Prac_08.car import Car
+
+
+class Taxi(Car):
+    """Specialised version of a Car that includes fare costs."""
+    price_per_km = 1.23
+
+    def __init__(self, name, fuel):
+        """Initialise a Taxi instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.current_fare_distance = 0
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return "{}, {}km on current fare, ${:.2f}/km".format(super().__str__(),
+                                                             self.current_fare_distance,
+                                                             self.price_per_km)
+
+    def get_fare(self):
+        """Return the price for the taxi trip."""
+        taxi_fare = (self.price_per_km * self.current_fare_distance)
```

Comment:
You could simplify this by changing it to 
fare = round(self.price_per_km * self.current_fare_distance, 1)
        return fare